sdk/formrecognizer/azure-ai-formrecognizer/swagger/README.md

```diff
@@ -63,7 +63,7 @@ directive:
 ### To run, use `autorest --tag:formrecognizer-documentanalysis README.md`
 ``` yaml $(tag) == 'formrecognizer-documentanalysis'
 use: '@autorest/java@4.1.13'
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/cognitiveservices/data-plane/FormRecognizer/stable/2023-07-31/FormRecognizer.json
+input-file: ./FormRecognizer.json
```

Comment:
Change it to the actual URL Before merging it to the main.

Reply:
we need it withthis for some changes